### DISCUSSION TITLE
id 검증에 대한 논의

### DISCUSSION THREAD

**Discussion by gkdud583** [prgrms-web-devcourse/Team-Saiko-BidMarket-BE#162] [Polls]
* 파라미터로 들어오는 id 값이 매번 양수인지 확인하는 코드가 테스트코드와 프로덕션 코드에 반복적으로 나오게 되는데 레이 말처럼 음수거나 0인 id는 애초에 db에 존재할수 없기 때문에 굳이 이부분을 매번 테스트 하지 않아도 될것 같다는 생각이 듭니다!
* 이부분에 대해 어떻게 생각하시나요?

**Comment by youngjunryu** [comment]
지우자
지우개

**Comment by UJ15** [comment]
이걸 외 않지워?

**Comment by dojinyou** [comment]
Assert문 말고도 constaraint로 불필요하다는 말씀이시죠??

**Reply by gkdud583** [reply]
넵! 그 말이었습니다!
